react/features/facial-recognition/actions.js

```diff
@@ -135,33 +159,46 @@ export function startFacialRecognition() {
         const { recognitionActive } = state['features/facial-recognition'];
 
         if (recognitionActive) {
+            logger.log('Face recognition already active.');
+
             return;
         }
-        const localVideoTrack = getLocalVideoTrack(state['features/base/tracks']);
+
+        const localVideoTrack = track || getLocalVideoTrack(state['features/base/tracks']);
```

Comment:
On firefox we don't remove tracks when muting, we just mark them as disabled. I wonder if we need an explicit check about it around here.

Reply:
but this function only runs when we start (e.g. unmute camera).